tests/oracle/test-oracle.cpp

```diff
@@ -136,8 +138,10 @@ TEST_CASE("Oracle blob", "[oracle][blob]")
             oracle_blob_backend *blobBackEnd
                 = static_cast<oracle_blob_backend *>(b.get_backend());
 
+            /*
```

Comment:
Let's just remove it if it's not needed?

Reply:
Yes that will be done during cleanup of my oracle commits.